runestone/common/runestonedirective.py

```diff
@@ -337,11 +337,11 @@ def add_skulpt_js(app):
     if first_time(app, "add_skulpt_js"):
         app.add_autoversioned_javascript("skulpt.min.js")
         app.add_autoversioned_javascript("skulpt-stdlib.js")
-        app.add_javascript("https://cdn.jsdelivr.net/npm/vega@4.0.0-rc.2/build/vega.js")
-        app.add_javascript(
+        app.add_js_file("https://cdn.jsdelivr.net/npm/vega@4.0.0-rc.2/build/vega.js")
```

Comment:
As skulpt is now bundled into runestone.js this probably isn't necessary.

Reply:
OK, great! I assume this means I can delete lines 340-346 from the PR, then?

Reply:
yes

Reply:
OK, done in d867c61b549fb6e59f101bac6b06ffd84012fadd.